src/script/conversation/ConversationRepository.ts

```diff
@@ -2100,7 +2100,7 @@ export class ConversationRepository {
         // We need to check if the conversation exists on backend
         await this.conversationService.getConversationById(inccessibleConversation);
       } catch (error) {
-        if (error instanceof ConversationError && error.type === ConversationError.TYPE.CONVERSATION_NOT_FOUND) {
+        if (isBackendError(error) && error.label === BackendErrorLabel.NO_CONVERSATION) {
```

Comment:
There is no guarantee that we will throw a custom error at this point. So, the solution is to check for a backend error and the desired label.